metadata/db.go

```diff
@@ -139,6 +140,13 @@ type db struct {
 }
 
 func NewDB(root string) (DB, error) {
+	if _, err := os.Stat(root); os.IsNotExist(err) {
+		if err := os.Mkdir(root, os.FileMode(0740)); err != nil {
```

Comment:
Curious why the odd permission? Typical directories are `0775`.

Reply:
No real reason, guess it's just a habit.